lib/publiccloud/instance.pm

```diff
@@ -346,15 +350,20 @@ sub wait_for_ssh
 
     script_run("ssh  -i /root/.ssh/id_rsa -v $args{username}\@$args{public_ip} true", timeout => 360);
     # Debug output: We have occasional error in 'journalctl -b' - see poo#96464 - this will be removed soon.
-    $self->run_ssh_command(cmd => 'sudo journalctl -b', proceed_on_failure => 1, username => $args{username});
+    # Exclude 'mr_test/saptune' test case as it will introduce random softreboot failures.
+    if (!get_var('PUBLIC_CLOUD_SLES4SAP')) {
+        $self->run_ssh_command(cmd => 'sudo journalctl -b', proceed_on_failure => 1, username => $args{username}, timeout => 360);
+    }
```

Comment:
I don't understand this.

Reply:
Thanks [grisu48](https://github.com/grisu48) , you are right this debug code introduces timeout. I had tried a lot on increasing the timeout, I remember I tried 90, 180,  even 3600, it is better but I can still hit the timeout see https://openqaworker15.qa.suse.cz/tests/103196#step/1_saptune_overrides/39.

Reply:
Thank you for the update. I still believe that we should understand why this issue arises here, despite a large timeout.

But as stated above, we can proceed in skipping this check for now on SAP, as it's not needed and should have been anyways only temporary.

LGTM!